FullStorySegmentMiddleware.podspec

```diff
@@ -126,6 +126,5 @@ Pod::Spec.new do |spec|
 
   # spec.xcconfig = { "HEADER_SEARCH_PATHS" => "$(SDKROOT)/usr/include/libxml2" }
   spec.dependency 'FullStory'
-  spec.dependency 'Analytics'
-
+  spec.dependency 'Analytics', '~> 4.1'
```

Comment:
Is there a reason we're now restricting to `4.x`?

Reply:
I was thinking that now we are always importing Segment module as `Segment` not `Analytics`, which would break this dependency < 4.1.
I was planning on adding the readme and changelog when releasing a new tag
I don't think there are breaking code changes, maybe we could use `canImport` in swift to check that? I wasn't entirely sure what's needed for swift but if that sounds like a way I will test it out